projects/lucene/project.yaml

```diff
@@ -6,6 +6,7 @@ fuzzing_engines:
 sanitizers:
   - address
 primary_contact:
+  - "david.w.smiley@gmail.com"
```

Comment:
The `primary_contact` is just a string and not a list, you do `primary_contact: "david.w.smiley@gmail.com"` instead?